### PR TITLE
Doc/generic jdbc connection

### PR DESCRIPTION
**Pull request by ElectricWound**
Guide to connect using any JDBC driver via a generic JdbcProfile implementation. The example uses a Databricks db.

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @ElectricWound,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[https://www.lightbend.com/contribute/cla](https://www.lightbend.com/contribute/cla)

**Comment by ElectricWound** [comment]
I signed the CLA

**Comment by nafg** [comment]
Thanks!

File: doc/paradox/database.md [hunk 1]
```diff
@@ -56,6 +56,28 @@ To use the MySQL driver, the following library dependency needs to be configured
...
+
+Here is an example for setting up a connection to a Databricks DB using the Databricks JDBC driver, depended on via SBT:
+```scala
+libraryDependencies += "com.databricks" % "databricks-jdbc" % "2.6.29"
+```
+
+To configure the connection a profile implementation is required. It is sufficient to extend the `JdbcProfile` trait with an
+empty object.
+
+@@snip [GenericJdbcProfile.scala](../code/GenericJdbcProfile.scala) { #genericJdbcProfile }
```
Comment (nafg) [inline doc/paradox/database.md:72]: CI shows that `paradoxMarkdownToHtml` is failing with:

```
[error] Label [genericJdbcProfile] not found in [/home/runner/work/slick/slick/doc/target/preprocessed/../code/GenericJdbcProfile.scala] at /home/runner/work/slick/slick/doc/target/preprocessed/database.md:72
[error] @@snip [GenericJdbcProfile.scala](../code/GenericJdbcProfile.scala) { #genericJdbcProfile }
[error] ^
```

**Review by nafg — CHANGES_REQUESTED** [review]

**Review by nafg — APPROVED** [review]

**Pull request merged**